apps/src/componentLibrary/tags/Tags.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import {ComponentSizeXSToL} from '@cdo/apps/componentLibrary/common/types';
+import Tag, {TagProps} from './_Tag';
+import moduleStyles from './tags.module.scss';
+
+export interface TagsProps {
+  /** Array of tags to be rendered */
+  tagsList: TagProps[];
```

Comment:
Wondering if it would be better to just have the children be of type `Tag` instead of a separate `tagsList` prop? Something like -
```
<Tags>
  <Tag label="label" tooltipContent="tooltip" ... />
  <Tag ... />
</Tags>
```
Though I can also understand if we want to discourage consumers from using the `Tag` component directly - but maybe in that case the `<Tag>` can check it's parent to ensure that it's a `<Tags>`?

Reply:
Gotcha that makes sense!